cmd/wsm/wsm.go

```diff
@@ -301,10 +304,53 @@ func DeployCmd() *cobra.Command {
 	return cmd
 }
 
+func openBrowser(url string) error {
+	var cmd string
+	var args []string
+
+	switch runtime.GOOS {
+	case "windows":
+		cmd = "cmd"
+		args = []string{"/c", "start"}
+	case "darwin":
+		cmd = "open"
+	default: // "linux", "freebsd", "openbsd", "netbsd"
+		cmd = "xdg-open"
+	}
+	args = append(args, url)
+	return exec.Command(cmd, args...).Start()
+}
+
+func ConsoleCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "console",
+		Run: func(cmd *cobra.Command, args []string) {
+			pwd, err := kubectl.GetSecret("wandb-password", "default")
+			if err != nil {
+				panic(err)
+			}
+
+			url := "http://localhost:8080/console/login?password=" + base64.StdEncoding.EncodeToString(pwd)
+
+			time.AfterFunc(500*time.Millisecond, func() {
```

Comment:
gives time for the port forward to start so that they don't see a "site can't be reached" message